Description: Disable warning_is_error that causes FTBFS because of 
 missing dependencies in ipython
Author: Alastair McKinstry <mckinstry@debian.org>
Bug-Origin: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=950101
Last-Updated: 2020-01-30
Forwarded: no

--- a/doc/conf.py
+++ b/doc/conf.py
@@ -35,6 +35,9 @@
 print("python exec:", sys.executable)
 print("sys.path:", sys.path)
 
+# See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=950101
+ipython_warning_is_error = False
+
 if "CONDA_DEFAULT_ENV" in os.environ or "conda" in sys.executable:
     print("conda environment:")
     subprocess.run([os.environ.get("CONDA_EXE", "conda"), "list"])
